packages/ember-routing/lib/location/api.js

```diff
@@ -192,15 +193,6 @@ export default {
     @since 1.4.0
   */
   _getHash: function () {
-    // AutoLocation has it at _location, HashLocation at .location.
-    // Being nice and not changing
-    var href = (this._location || this.location).href;
-    var hashIndex = href.indexOf('#');
-
-    if (hashIndex === -1) {
-      return '';
-    } else {
-      return href.substr(hashIndex);
-    }
+    return getHash(this._location || this.location);
```

Comment:
I don't think we need to deprecate, it's private and I can just update. Not sure why that wasn't done before.